server/main.lua

```diff
@@ -176,31 +195,52 @@ QBCore.Commands.Add("drivinglicense", "Give a drivers license to someone", {{"id
                         SearchedPlayer.PlayerData.metadata["licences"]["driver"] = true
                         SearchedPlayer.Functions.SetMetaData("licences", SearchedPlayer.PlayerData.metadata["licences"])
                         TriggerClientEvent('ox_lib:notify', source, {
-                            title = 'Alert',
-                            description = "You have passed! Pick up your drivers license at the town hall",
-                            type = 'success'
+                            id = 'test_passed',
+                            description = 'You have passed! Pick up your drivers license at the town hall',
+                            style = {
+                                backgroundColor = '#141517',
+                                color = '#ededed'
+                            },
+                            icon = 'check',
+                            iconColor = '#27AE60'
                         })
                         TriggerClientEvent('ox_lib:notify', source, {
-                            title = 'Alert',
-                            description = ("Player with ID %s has been granted access to a driving license"):format(SearchedPlayer.PlayerData.source),
-                            type = 'success'
+                            id = 'license_granted',
+                            description = ("Player with ID %s has been granted access to a driving license"):format(SearchedPlayer
+                                .PlayerData.source),
+                            style = {
+                                backgroundColor = '#141517',
+                                color = '#ededed'
+                            },
+                            icon = 'check',
+                            iconColor = '#27AE60'
                         })
                         break
                     end
                 end
             end
         else
             TriggerClientEvent('ox_lib:notify', source, {
-                title = 'Alert',
-                description = "Can't give permission for a drivers license, this person already has permission",
-                type = 'error'
+                id = 'has_license',
+                description = 'Can\'t give permission for a drivers license, this person already has permission',
+                style = {
+                    backgroundColor = '#141517',
+                    color = '#ededed'
+                },
+                icon = 'check',
+                iconColor = '#27AE60'
             })
         end
     else
         TriggerClientEvent('ox_lib:notify', source, {
-            title = 'Alert',
-            description = "Player Not Online",
-            type = 'error'
+            id = 'not_online',
+            description = 'Player Not Online',
+            style = {
+                backgroundColor = '#141517',
+                color = '#ededed'
+            },
+            icon = 'check',
+            iconColor = '#27AE60'
```

Comment:
Consider extracting the Trigger ox_lib:notify call into a local function since the only thing changing is the description and id, you can reduce duplicate code.

Reply:
Alternatively, consider extracting the fields into constants in this file. Although I think the function would be cleaner.

Reply:
This is a decent suggestion, it definitely would improve the flow of the code by reducing chunky reiterations of the same code.